js/blocks/components/fields.js

```diff
@@ -1,8 +1,8 @@
 /**
  * WordPress dependencies
  */
-const { applyFilters } = wp.hooks;
-const { select } = wp.data;
+import { applyFilters } from '@wordpress/hooks';
```

Comment:
The reason for importing these as modules instead of using something like `wp.hooks` is so that we can control the version.

For example, `InspectorControls` and `InspectorAdvancedControls` were moved from `wp.editor` to `wp.block-editor`. And `wp.nux` was deprecated entirely.

If we control the versions of the modules, we should be less dependent on the version on Core and Gutenberg users have.

Reply:
A user reported an error from `wp.nux.DotTips` being deprecated in Gutenberg. I probably would have seen that if we imported that deprecated module.